Anonymization Proxy/anonymization-proxy.md

```diff
@@ -1,39 +1,40 @@
-## Overview
+## Anonymization Proxy
 
-Anonymization Proxy detects and anonymizes Personally Identifiable Information (PII) that originates from the use of ACE Knowledge Widgets (e.g., performing a search, chatting with an ACE Knowledge Chat Bot, etc.).
+Anonymization Proxy is a tool that detects and anonymizes Personally Identifiable Information (PII) originating from the use of ACE Knowledge Widgets. This tool is designed to help protect user privacy when interacting with ACE Knowledge Widgets. Any detected PII is processed within Telia cloud and never reaches foreign entities such as Microsoft Azure, where ACE Knowledge backend is hosted.
 
 ## Features
 
-* Anonymization of PII such as personal names, social security numbers, telephone numbers.
-* Client IP masking - IP address of the user that works with ACE Knowledge Widgets is automatically masked as it is only visible to Anonymization Proxy application. Anonymization Proxy does not store this information and does not forward it to ACE Knowledge backend.
+Anonymization Proxy offers the following features:
+  * Anonymization of PII such as personal names, social security numbers, and telephone numbers.
+  * Client IP masking - IP addresses are automatically masked as they are only visible to Anonymization Proxy application. Anonymization Proxy does not store this information and does not forward it to ACE Knowledge backend.
 
-## How can I use Anonymization Proxy?
+## Using Anonymization Proxy
 
-In order to be able to use Anonymization Proxy, please consider the following:
-  * Anonymization Proxy currently only works with public Widgets
-  * Since anonymization of PII is done outside of ACE Knowledge backend, it means that it is no longer possible to search on PII in Widgets when using Anonymization Proxy
-  * It is not possible to use Anonymization Proxy for v1-v3 Widget implementations
-  * Widgets distributed through Web provisions cannot be configured to use Anonymization Proxy
+Before using Anonymization Proxy, please note the following:
+  * Anonymization Proxy currently only works with public Widgets.
+  * Anonymization Proxy does not allow searching on PII in Widgets.
+  * Anonymization Proxy cannot be used for v1-v3 Widget implementations.
```

Comment:
I think we could reformulate this as a positive message. Something like "AP is applicable to Widgets v4 and OneWidget. Versions v1-v3 are not supported."